src/om_bootstrap/nav.cljs

```diff
@@ -169,15 +169,21 @@
       (u/clone-with-props child prop-fn))))
 
 (defn render-navbar-child [owner child bs]
-  (let [f (fn [props]
+  (let [changing? (om/get-state owner :changing?)
```

Comment:
looks busted. You're capturing a single value of `changing?` and closing it inside the function - so this'll never change.